api/routes/roles/index.js

```diff
@@ -1,5 +1,7 @@
 const get = require('./get');
+const put = require('./put');
 
-module.exports = (app, getEndpoint = get) => {
+module.exports = (app, getEndpoint = get, postEndpoint, putEndpoint = put) => {
```

Comment:
`postEndpoint` is placeholder i assume

Reply:
Yeah.  The `post` code is already written, I just to rebase and finish up the tests.